### PR TITLE
deprecate enhance print 

### PR DESCRIPTION
**Pull request by saartochner**

### CODE REVIEW

File: src/test/unit/test_tracer.py [hunk 1]
```diff
@@ -183,28 +181,26 @@ def lambda_test_function(event, context):
...
+        assert any(line == "hello" for line in capturer.get_lines())
 
 
-def test_wrapping_without_print_override(context):
-    @lumigo_tracer()
+def test_wrapping_enhanced_print_backward_compatible(context):
+    @lumigo_tracer(enhance_print=True)
     def lambda_test_function(event, context):
         print("hello")
+        assert Configuration.enhanced_print is False
```
Comment (nirLumigo) [inline src/test/unit/test_tracer.py:199]: I think that `Configuration` shouldn't have the `enhanced_print` property anymore, isn't it?

**Review by nirLumigo — CHANGES_REQUESTED** [review]
Nice :)

File: src/test/unit/test_tracer.py (hunk 1 above)
Comment (saartochner) [reply src/test/unit/test_tracer.py]: I agree. Deleted :)

**Review by nirLumigo — APPROVED** [review]

**Comment by saartochner** [comment]
Opened an identical PR under different name (ci purpose)

**Pull request closed without merging**